arches/app/src/components/ControlledListManager/ControlledListSplash.vue

```diff
@@ -0,0 +1,32 @@
+<script setup lang="ts">
+import { useGettext } from "vue3-gettext";
+
+const props: { description: string } = defineProps(["description"]);
```

Comment:
does the description need to be a prop?

Reply:
`<ControlledListTable>` provides "select a list from the sidebar", and `<ControlledListEditor>` provides "select an item from the sidebar".